web-ui/src/main/resources/catalog/style/gn_icons.less

```diff
@@ -142,35 +142,35 @@
   content: @fa-var-fighter-jet;
 }
 .gn-icon-inlandWaters:before {
-  content: @fa-var-tint;
+  content: @fa-var-bridge-water;
 }
 .gn-icon-location:before {
   content: @fa-var-map-marker;
 }
 .gn-icon-oceans:before {
-  content: @fa-var-anchor;
+  content: @fa-var-water;
 }
 .gn-icon-planningCadastre:before {
   content: @fa-var-home;
 }
 .gn-icon-society:before {
-  content: @fa-var-comments;
+  content: @fa-var-city;
 }
 .gn-icon-structure:before {
   content: @fa-var-building;
 }
 .gn-icon-transportation:before {
-  content: @fa-var-truck;
+  content: @fa-var-truck-plane;
 }
 .gn-icon-utilitiesCommunication:before {
-  content: @fa-var-phone;
+  content: @fa-var-satellite-dish;
```

Comment:
You can also add extraTerrestrial and disaster which are in ISO19115-3

Reply:
@fxprunayre where can I find the exact name for these two?

Is it
- gn-icon-extraTerrestrial
- gn-icon-disaster

Reply:
Yes it is the ISO enumeration key

Reply:
I added 2 new icons for the topics